CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## v0.7.3 (2021-12-14)
+  - API:
+    - `view_index` builtin (missing from v0.7)
```

Comment:
can we remove it from 0.7.0 list of changes? and also remove the "(missing from)" piece